### PR TITLE
feat: Navbar component added

### PR DESCRIPTION
**Pull request by mehulsatardekar**
files to check
src/components/navbar/Navbar.tsx

### CODE REVIEW

File: src/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          
+          if(error) throw error;
+          navigate('/');
+         // window.location.reload();
+        }catch(error){
+            notifyError('oops Some error occured while signing out')
+            console.error(error);
+        }
+    }
+    //console.log(currentUser);
```
Comment (Naman-Saxena1) [inline src/components/navbar/Navbar.tsx]: Can remove commented code

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
It is a small PR, code quality looks good and follows best practices 👍

File: src/components/navbar/Navbar.tsx (hunk 1 above)
Comment (mehulsatardekar) [reply src/components/navbar/Navbar.tsx]: sure @Naman-Saxena1

**Review by rahulyadav139 — APPROVED** [review]
Appreciation: I didn't find anything to correct. The codes are looking great and readable. Great work.

**Pull request merged**